### DISCUSSION TITLE
Can you help me with your star please?⭐ 💕

### DISCUSSION THREAD

**Discussion by JaimeTR** [community/community#28642] [Other Feature Feedback, Questions, & Ideas]
Hello, can you help me by giving stars ⭐ to my repository:

◉ https://github.com/JaimeTR/SvenBot  or 
◉ https://github.com/JaimeTR/instahack  

I want to have the GitHub badges, in the same way I will help those who support me.💕

**Comment by augustoaraujoo** [comment]
ok :)
